ipaserver/install/bindinstance.py

```diff
@@ -775,6 +776,8 @@ def __setup_sub_dict(self):
             NAMED_PID=paths.NAMED_PID,
             NAMED_VAR_DIR=paths.NAMED_VAR_DIR,
             BIND_LDAP_SO=paths.BIND_LDAP_SO,
+            DATA_DIR='' if ipaplatform.NAME == 'debian' else 'data/',
+            DISABLE_DEBIAN='//' if ipaplatform.NAME == 'debian' else '',
             )
```

Comment:
I don't like platform dependent code outside of `ipaplatform` module, I prefer to move this logic into tasks, or even better to have template files per platform if needed.

Reply:
I like Martin's suggestion to move the variables to tasks.

I'd rename DISABLE_DEBIAN to DISABLE_ROOTS or something more generic. Maintaining two files could be prone to error especially since the diffs at the moment seem to be relatively small.